goinstant/prompts.go

```diff
@@ -81,57 +180,382 @@ func selectPackageDocker() {
+		RunDirectDockerCommand([]string{"docker", "elastic-analytics", "init"})
+		selectDefaultInstall()
+
+	case "Initialise Elastic-Pipeline":
+		fmt.Println("...Setting up Elastic-Pipeline Package")
+		RunDirectDockerCommand([]string{"docker", "elastic-pipeline", "init"})
+		selectDefaultInstall()
+
+	case "Initialise Electronic Medical Record":
+		fmt.Println("...Setting up Electronic Medical Record Package")
+		RunDirectDockerCommand([]string{"docker", "emr", "init"})
+		selectDefaultInstall()
+
+	case "Initialise Health Management Information System":
+		fmt.Println("...Setting up Health Management Information System Package")
+		RunDirectDockerCommand([]string{"docker", "hmis", "init"})
+		selectDefaultInstall()
+
+	case "Initialise Health Worker":
+		fmt.Println("...Setting up Health Worker Package")
+		RunDirectDockerCommand([]string{"docker", "healthworker", "init"})
+		selectDefaultInstall()
+
+	case "Initialise Facility Registry":
 		fmt.Println("...Setting up Facility Registry Package")
-		RunDirectDockerCommand([]string{"docker", "facility", "up"})
-		selectPackageDocker()
+		RunDirectDockerCommand([]string{"docker", "facility", "init"})
+		selectDefaultInstall()
 
-	case "Launch Workforce":
+	case "Initialise Workforce":
 		fmt.Println("...Setting up Workforce Package")
-		RunDirectDockerCommand([]string{"docker", "healthworker", "up"})
-		selectPackageDocker()
+		RunDirectDockerCommand([]string{"docker", "healthworker", "init"})
+		selectDefaultInstall()
+
+	case "Stop All Services and Cleanup Docker":
+		fmt.Println("Stopping and Cleaning Up Everything...")
+		RunDirectDockerCommand([]string{"docker", "destroy"})
+		selectDefaultInstall()
 
 	case "Stop and Cleanup Core":
 		fmt.Println("Stopping and Cleaning Up Core...")
 		RunDirectDockerCommand([]string{"docker", "core", "destroy"})
-		selectPackageDocker()
+		selectDefaultInstall()
+
+	case "Stop and Cleanup Client":
+		fmt.Println("Stopping and Cleaning Up Client...")
+		RunDirectDockerCommand([]string{"docker", "client", "destroy"})
+		selectDefaultInstall()
+
+	case "Stop and Cleanup Elastic-Analytics":
+		fmt.Println("Stopping and Cleaning Up Elastic-Analytics...")
+		RunDirectDockerCommand([]string{"docker", "elastic-analytics", "destroy"})
+		selectDefaultInstall()
+
+	case "Stop and Cleanup Elastic-Pipeline":
+		fmt.Println("Stopping and Cleaning Up Elastic-Pipeline...")
+		RunDirectDockerCommand([]string{"docker", "elastic-pipeline", "destroy"})
+		selectDefaultInstall()
+
+	case "Stop and Cleanup Electronic Medical Record":
+		fmt.Println("Stopping and Cleaning Up Electronic Medical Record...")
+		RunDirectDockerCommand([]string{"docker", "emr", "destroy"})
+		selectDefaultInstall()
+
+	case "Stop and Cleanup Health Management Information System":
+		fmt.Println("Stopping and Cleaning Up Health Management Information System...")
+		RunDirectDockerCommand([]string{"docker", "hmis", "destroy"})
+		selectDefaultInstall()
+
+	case "Stop and Cleanup Health Worker":
```

Comment:
Is this not duplicating the logic under "Stop and Cleanup Workforce"?

Reply:
Health Workforce and Health Worker are different components

Reply:
Indeed, but both have exactly the same docker command, so what am I missing here? 😁

Reply:
Indeed, so they should have different docker commands right? 😁

Reply:
lol, I didn't expand the whole file :p thought you were confused

Reply:
Cool, thanks for correcting it 🙂